test/runtests.jl

```diff
@@ -4,7 +4,7 @@ using Pkg
 # add packages
 to_add = [
     PackageSpec(name="Plots"),
-    PackageSpec(name="StatPlots"),
+    PackageSpec(name="StatsPlots"),
```

Comment:
Hilariously you spelled it wrong in the commit message :-)

Reply:
Hey, I got it right where it counts. Good eye!